test/lib/log.js

```diff
@@ -1,7 +1,27 @@
 /* Any copyright is dedicated to the Public Domain.
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
-var noop = function () {}
+const assert = require('assert');
+
+// all the promise wrapping in tests makes it unclear who actually 
+// called mozlog incorrectly. So, record this location directly.
+function getMozlogCallerLocation() {
+  var err = new Error('getMozlogCallerLocation').stack.split('\n')[3]
+  return err
+}
+
+// no-op except check that first arg is a string, etc., a la mozlog
+var noop = function () {
```

Comment:
I like this function so much, I wonder if it is worth exporting from mozlog itself somehow, so that other repos can benefit (not for this PR of course, I mean later). I guess the `getMozlogCallerLocation` implementation may be repo-specific, but we could just pass that in.

If anyone else thinks that is a good idea, I'll open an issue on mozlog.